pkg/skaffold/survey/survey.go

```diff
@@ -63,21 +58,25 @@ func New(configFile string) *Runner {
 
 func (s *Runner) DisplaySurveyPrompt(out io.Writer) error {
 	if isStdOut(out) {
-		output.Green.Fprintf(out, Prompt)
+		output.Green.Fprintf(out, hats.prompt())
```

Comment:
Add a TODO here?

(It seems funny to do everything but actually pull in the find-relevant survey logic!)

Reply:
yes. Was going to implement it later as per the implementation plan.